src/config.ts

```diff
@@ -1,10 +1,10 @@
-import { version } from '../package.json'
+import packageInfo from '../package.json'
```

Comment:
I got a warning about this, so I fixed it as a drive-by

`Should not import the named export 'version' (imported as 'version') from default-exporting module (only default export is available soon)`